cmd/backup/main.go

```diff
@@ -4,19 +4,32 @@
 package main
 
 import (
+	"flag"
 	"fmt"
+	"log"
 	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/robfig/cron/v3"
 )
 
-func main() {
-	s, err := newScript()
+func runBackup(c *Config) *error {
```

Comment:
`error' is an interface type

```go
// The error built-in interface type is the conventional interface for
// representing an error condition, with the nil value representing no error.
type error interface {
	Error() string
}
```

so it's rather uncommon to return a pointer, else you might end up with a pointer to a pointer.

I.e. this can be `func runBackup(c *Config) error`

Reply:
I already changed it, the reason I had to was due to the defered unlock call which itself could return an error.

Reply:
I decided to make #341 where the logging could be standardized and improved in the whole application. Right now the `main` loop will use the logger initialized at startup.